src/dymaptic.GeoBlazor.Core/Components/Views/MapView.razor.cs

```diff
@@ -2062,6 +2094,7 @@ protected override async Task OnInitializedAsync()
     protected override async Task OnAfterRenderAsync(bool firstRender)
     {
         ApiKey = Configuration["ArcGISApiKey"];
+        AppId = Configuration["ArcGISAppId"];
```

Comment:
Is this safe if either or both are not specified in the configuration?

Reply:
Yes, the `IConfiguration` indexer is defined to return a nullable string if the value isn't found.